eclipse/plugins/org.switchyard.tools.ui/src/org/switchyard/tools/ui/common/SwitchYardSettingsGroup.java

```diff
@@ -805,7 +805,15 @@ private void populateRuntimeVersionsList(IProgressMonitor monitor) {
         }
         // add default version
         final ArtifactVersion defaultVersion = parseVersion(NewSwitchYardProjectWizard.DEFAULT_RUNTIME_VERSION);
-        _availableVersions.add(defaultVersion);
+        if (!_availableVersions.contains(defaultVersion)) {
+            _availableVersions.add(defaultVersion);
+        }
+
+        // for SWITCHYARD-2959 - add previous version as well 
```

Comment:
I prefer not setting JIRA number, would be better to extract a method with a name like ensureDefaultAndPreviousVersionAvailable